### PR TITLE
Move regression test to the end

### PR DESCRIPTION
**Pull request by szy21**
## Content

----
- [ ] I have read and checked the items on the review checklist.

### CODE REVIEW

**Comment by szy21** [comment]
@charleskawczynski I think the multiple command doesn’t work well with multi-line command for TC?

File: .buildkite/pipeline.yml [hunk 1]
```diff
@@ -501,6 +523,7 @@ steps:
           --moist equil
           --job_id edmf_soares
           --dt 0.25secs
+          - julia --color=yes --project=examples regression_tests/test_mse.jl --job_id edmf_soares --out_dir edmf_soares
```
Comment (charleskawczynski) [inline .buildkite/pipeline.yml:526]:
```suggestion
        command: julia --color=yes --project=examples regression_tests/test_mse.jl --job_id edmf_soares --out_dir edmf_soares
```
What if we did this instead?

**Pull request merged**